ui/packages/ui/webpack.common.js

```diff
@@ -39,7 +42,6 @@ module.exports = (argv) => {
         {
           test: /\.css$/,
           use: [
-            // MiniCssExtractPlugin.loader, 
```

Comment:
I had intentionally left this commented to give the bug reproduction scenario. here https://github.com/patternfly/patternfly-react/issues/5726

Reply:
Added it back.